components/src/dynamo/planner/plugins/orchestrator/orchestrator.py

```
@@ -0,0 +1,375 @@
+                    plugin_type=entry.plugin_type,
+                    priority=entry.priority,
+                    endpoint=entry.endpoint,
+                    auth_token=entry.auth_token,
+                    protocol_version=entry.protocol_version,
+                    version=entry.version,
+                    execution_interval_seconds=entry.execution_interval_seconds,
+                    hold_policy=entry.hold_policy,
+                    needs=list(entry.needs),
+                )
+                resp = await self._registry.register(req)
+            except Exception as exc:
```

Ruff (0.15.14)

[warning] Do not catch blind exception: `Exception`

(BLE001)